### ISSUE TITLE
Getting 503 for calls from other devices from "outside"

### ISSUE THREAD

**Issue by thaeger71** [RestComm/Restcomm-Connect#973]
If I register a user from the same IP of the restcomm-server, let's say 172.20.9.1 then I'am able to call all the test numbers like 1234 which just playing an audio message.
If I register from outside, a different IP address, it doesn't work. Then I get:
"503 Problem to setup services 1234"
Reason: Exception,while,trying,to,download,RCML

What could be here the problem ?

**Comment by deruelle** [comment]
Please provide DEBUG Logs for both Restcomm and Media Server as well as network trace (tcpdump). Please also provide the command you used to start RestComm as well as restcomm.conf

**Comment by thaeger71** [comment]
Hey Jean,

you're right.

Here the command for starting the standalone version:

./bin/standalone.sh -b 172.20.9.1

Command for starting mms:

./bin/run.sh

See attached the logfiles, configfiles and the tcpdump.

[issue-973.zip](https://github.com/RestComm/Restcomm-Connect/files/197537/issue-973.zip)

**Comment by deruelle** [comment]
hi @thaeger71, this is not the correct way to start restcomm, please read http://docs.telestax.com/restcomm-quick-start-user-guide/

**Comment by thaeger71** [comment]
ok, I've done so now, but no difference ...

**Comment by deruelle** [comment]
Can you reattach the logs for the docker run ?

**Comment by thaeger71** [comment]
Hi guys,
today I had time to setup a new fresh docker image, I copied my changed jar's into the restcomm/WEB-INF/lib folder, registered my external devices and suddenly it worked. I've tried that before and it didn't work....maybe something has changed in the right way by accident, so that I can say the current version of the docker image combined with the changes of issue 935/970 also external devices can rigister and cann connect to an app and hear audio.
This isuue can be closed so far.
